city_scrapers/spiders/losca_Health_Commission.py

```diff
@@ -0,0 +1,104 @@
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class LoscaHealthCommissionSpider(CityScrapersSpider):
+    name = "losca_Health_Commission"
+    agency = "Los Angeles Health Commission"
+    timezone = "America/Los_Angeles"
+    committee_id: int = 6
+    start_year: int = 2024
+    default_location = {
+        "address": "200 N Spring St, Room 340 (CITY HALL) Los Angeles, CA 90012",
+        "name": "Los Angeles City Health Commission",
+    }
+    start_urls = [
+        f"https://lacity.primegov.com/api/v2/PublicPortal/ListArchivedMeetingsByCommitteeId?year={start_year}&committeeId={committee_id}"  # noqa
+    ]
+
+    def parse(self, response):
+        """
+        This spider retrieves meetings for the specified `start_year`.
+        Update `start_year` to change the target year.
+        """
+        try:
+            items = response.json()
+            if not isinstance(items, list):
+                self.logger.error(f"Expected list of items, got {type(items)}")
+                return
+
+            for item in items:
+                required_fields = ["title", "dateTime", "videoUrl", "documentList"]
+                if not all(field in item for field in required_fields):
+                    missing_fields = [
+                        field for field in required_fields if field not in item
+                    ]
+                    self.logger.warning(
+                        f"Missing required fields {missing_fields} in item with id {item.get('id', 'N/A')}"  # noqa
+                    )
+                    continue
+
+                meeting = Meeting(
+                    title=self._parse_title(item),
+                    description="",
+                    classification=COMMISSION,
+                    start=self._parse_start(item),
+                    end=None,
+                    all_day=False,
+                    time_notes="",
+                    location=self.default_location,
+                    links=self._parse_links(item),
+                    source=response.url,
```

Comment:
For `Meeting.source`, website URL is preferable over API URL. So in this case it can be a hardcoded value "https://clerk.lacity.gov/clerk-services/council-and-public-services/city-health-commission/commission-meetings"